include/dlt/dlt_common.h

```diff
@@ -377,6 +377,11 @@ enum {
  */
 #   define DLT_LINE_LEN 1024
 
+/**
+ * DLT fifo basedir length
+ */
+#define DLT_BASEDIR_MAX PATH_MAX - 16
```

Comment:
Where did 16 come from? Either having another macro or give some explanation would be helpful.